clients/include/hipblaslt_common.yaml

```diff
@@ -85,6 +91,11 @@ Real precisions 1 bytes dst fp16: &real_precisions_1b_dst_f16
   - *bf8f8_precision_dst_fp16
   - *f8bf8_precision_dst_fp16
 
+Real precisions 1 bytes dst bf16: &real_precisions_1b_dst_bf16
+  - *f8_precision_dst_bf16
+  - *f8bf8_precision_dst_bf16
+  # - *bf8f8_precision_dst_bf16 # TODO
+
 Real precisions 1 bytes dst fp8: &real_precisions_1b_dst_f8
```

Comment:
the third one is not included in this pr but we are gonna support it later. so this is just a placeholder